paddle/contrib/inference/paddle_inference_api_impl.cc

```diff
@@ -135,13 +135,14 @@ bool PaddlePredictorImpl::Run(const std::vector<PaddleTensor> &inputs,
 
 std::unique_ptr<PaddlePredictor> PaddlePredictorImpl::Clone() {
   VLOG(3) << "Predictor::clone";
-  std::unique_ptr<PaddlePredictor> cls(new PaddlePredictorImpl(config_));
```

Comment:
this will make mac success but make manylinux fail.

try return pointer, that mac works and might make manylinux success.

Reply:
There are many codes in fluid return a unique_ptr without the problem.  Not sure why this place face problem on manylinux build.